microsoft-edge/devtools-guide-chromium/reasons-edge-devtools.md

```diff
@@ -0,0 +1,215 @@
+
+
+### Open and close tools easily
+
+By clicking the **More Tools** (![More Tools icon.](reasons-edge-devtools-images/more-tools-icon.png)) button on the main toolbar or the Drawer toolbar, you can add tools without having to use the **Customize and control DevTools** `...` menu.  You can click the **x** button on any tool tab to remove the tool.  The **Elements**, **Sources** and **Console** tools are always open, and don't have an **x** on their tab.
+
+Adding a new tool to a menu bar by clicking the **More Tools** (![More Tools icon.](reasons-edge-devtools-images/more-tools-icon.png)) button:
+
+![Clicking the + button to add a tool.](reasons-edge-devtools-images/more-tools-button-adds-tool-on-toolbar.png)
+
+Removing a tool by clicking the **x** button on the tab of the tool:
+
+![Removing a tool by activating the x button on the tab of the tool](reasons-edge-devtools-images/click-tab-x-button-to-remove-tool.png)
+
+See [The More Tools menus](about-tools.md#the-more-tools-menus) in _About the list of tools_.
+
+
+### Move tools around in the UI
+ 
+By right-clicking any tab in the main toolbar or Drawer toolbar, you can move a tool from the top to the bottom panel, and vice versa.
+
+Moving a tool to the bottom drawer using the right-click menu:
+
+![Moving a tool to the bottom drawer using the right-click menu.](reasons-edge-devtools-images/right-click-tool-tab-move-drawer-toolbar.png)
+
+The tool now appears on the bottom drawer:
+
+![The tool now appears on the bottom drawer.](reasons-edge-devtools-images/tool-tab-moved-bottom-toolbar.png)
+
+See [Panel tools vs. Drawer tools](about-tools.md#panel-tools-vs-drawer-tools) in _About the list of tools_.
+
+
+### Focus Mode with Activity Bar and Quick View tool menu
+
+**Focus Mode** makes the DevTools UI even easier to use.  The Microsoft Edge DevTools team learned through user interviews and feedback sessions that the number of tools displayed at a given time can be overwhelming.  This is why we introduced Focus Mode, which groups a large set of features for related tasks into various tool panels, and then allows you to show only the tools that you need for your current activity or work mode.
+
+Focus mode showing only tools related to Elements debugging:
+
+![Focus mode showing only tools related to Elements debugging.](reasons-edge-devtools-images/focus-mode-with-only-elements-debugging-tools.png)
+
+Focus mode showing only tools related to Source editing:
+
+![Focus mode showing only tools related to Source editing.](reasons-edge-devtools-images/focus-mode-with-only-source-editing-tools.png)
+
+See [Simplify DevTools using Focus Mode](experimental-features/focus-mode.md).
```

Comment:
I think we need to rephrase this. This was the first design of Focus Mode (where multiple tools would be grouped together under one icon), but it no longer works this way.
I would advise using just 1 image, and changing the description of the feature to something similar to what we have at the top of https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/experimental-features/focus-mode

Rachel should review this part too.